### ISSUE TITLE
Don't return from methods that definitely return `undefined`

### ISSUE THREAD

**Issue by GoodForOneFare** [Shopify/javascript#115]
The Coffee=>JS conversion adds a bunch of meaningless return values.  We can reduce copy+paste anti-patterns by removing some of them.  

If they leave behind meaningless returns, we have other transforms to clean that up.

Likely candidates:
* `event.preventDefault()`
* `console.log`
* `console.warn`
* `_.forEach`, `_.each`
* `window.postMessage`

```
// Before
function foo(event) {
  return event.preventDefault();
}

// After
function foo(event) {
  event.preventDefault();
  return;
}
```

**Comment by fandy** [comment]
Fixed in #171

**Comment by fandy** [comment]
Closed the wrong issue :(
